docs/RELEASE-NOTES.rst

```diff
@@ -7,7 +7,24 @@ Release Notes for BIG-IP Controller for Kubernetes
 Added Functionality
 ```````````````````
 * Added route domain support for the VxLAN forwarding data base addresses (FDB).
-  Also, Removed the restriction on setting the default route domain
+  Also, removed the restriction on setting the default route domain
+* Flannel VxLAN support. You can now run the controller in Kubernetes in cluster mode
+  with a Flannel VXLAN tunnel specified, and the controller will populate FDB and ARP entries
+  on the BIG-IP.
+* Ingresses will now share a virtual server if specified with the same IP address and port.
+* Accepts a default IP address parameter and uses it as the VIP for all Ingresses
+  with the annotation 'virtual-server.f5.com/ip="controller-default"'.
+* Health monitors for ConfigMaps, Ingresses, and Routes support recv strings.
+* Version info added to the container and logs
+* UDP support for ConfigMaps
+
+Bug Fixes
+`````````
+* HTTPS redirect now applies to individual Routes instead of all Routes.
```

Comment:
If we have github issues for these bugfixes, can we add a link to those (I think we've done this in previous releases).